addons/website_hr_recruitment/tests/test_website_hr_recruitment.py

```diff
@@ -1,17 +1,18 @@
-from openerp.api import Environment
-import openerp.tests
-@openerp.tests.common.at_install(False)
-@openerp.tests.common.post_install(True)
-class TestWebsiteHrRecruitmentForm(openerp.tests.HttpCase):
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import odoo.tests
+
+@odoo.tests.common.at_install(False)
+@odoo.tests.common.post_install(True)
+class TestWebsiteHrRecruitmentForm(odoo.tests.HttpCase):
     def test_tour(self):
         self.phantom_js("/", "odoo.__DEBUG__.services['web.Tour'].run('website_hr_recruitment_tour', 'test')", "odoo.__DEBUG__.services['web.Tour'].tours.website_hr_recruitment_tour")
 
         # get test cursor to read from same transaction browser is writing to
         cr = self.registry.cursor()
         assert cr == self.registry.test_cr
-        env = Environment(cr, self.uid, {})
 
-        record = env['hr.applicant'].search([('description', '=', '### HR RECRUITMENT TEST DATA ###')])
+        record = self.env(cr=cr)['hr.applicant'].search([('description', '=', '### HR RECRUITMENT TEST DATA ###')])
```

Comment:
It will allow us to use the environment of self with different cursor. No need to define whole new Environment object.